### PR TITLE
GHA: sync README.md to Docker Hub

### PR DESCRIPTION
**Pull request by Al2Klimov**

### CODE REVIEW

**Comment by Al2Klimov** [comment]
## Before

Bildschirmfoto 2022-07-19 um 10 20 42

## After

Bildschirmfoto 2022-07-19 um 11 00 44

File: .github/workflows/sync-readme.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      - master
+
+jobs:
+  sync:
+    name: Sync README
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout HEAD
+        uses: actions/checkout@v1
```
Comment (julianbrost) [inline .github/workflows/sync-readme.yml:15]: I mean sure, `v1` will do the job, but why not use the latest available version?

Comment (Al2Klimov) [reply .github/workflows/sync-readme.yml:15]: It's a C&P ~error~.

Comment (julianbrost) [reply .github/workflows/sync-readme.yml:15]: So given that there were no changes and you just re-requested review, you want to keep it this way?

Comment (Al2Klimov) [reply .github/workflows/sync-readme.yml:15]: Yes.

Comment (julianbrost) [reply .github/workflows/sync-readme.yml:15]: Whatever.

**Review by julianbrost — APPROVED** [review]

**Pull request merged**